backend/graph/api/items.resolvers.go

```diff
@@ -134,7 +142,11 @@ func (r *showResolver) Image(ctx context.Context, obj *model.Show, style *model.
 	if style != nil && style.IsValid() {
 		s = style.String()
 	}
-	return e.Images.GetDefault(user.GetLanguagesFromCtx(ginCtx), s), nil
+	img := e.Images.GetDefault(user.GetLanguagesFromCtx(ginCtx), s)
+	if img == nil && e.Image.Valid {
+		img = &e.Image.String
+	}
+	return img, nil
```

Comment:
Would this bit of code (I see at least 3 repetitions) deserve to be packed into a named function? I think I know what it does but it would make it easier if it had a name directly on the struct. 

But if it's not "our" struct, don't bother.